pysindy/feature_library/weak_pde_library.py

```diff
@@ -0,0 +1,827 @@
+from itertools import combinations
+from itertools import combinations_with_replacement as combinations_w_r
+from itertools import product as iproduct
+
+import numpy.random
+from numpy import array
+from numpy import concatenate
+from numpy import empty
+from numpy import floor
+from numpy import isscalar
+from numpy import linspace
+from numpy import meshgrid
+from numpy import prod
+from numpy import reshape
+from numpy import shape
+from numpy import sum as np_sum
+from numpy import take
+from numpy import transpose
+from numpy import zeros
```

Comment:
I don't expect you to do this, because it'd be a pain, but if we ever refactor weak_pde_library we should switch to an `import numpy as np`. Importing individual numpy methods like this makes more sense when you're only using one or two methods.

Reply:
sounds good, this is an easy fix